src/utils/style-formatter.js

```diff
@@ -32,8 +32,7 @@ const formatColorProperty = (path, inputColor, defaultColor, isExpression) => {
 
 export default {
   getStyles(style, disabled, Theme) {
-    // TODO: use constants for default values?
-    let styles = 'width: 100%;height: 100%;font-weight: bold;border:none;';
+    let styles = 'width: 100%;height: 100%;border: none;padding: 4px;';
```

Comment:
Why do we need the padding here?

Reply:
![Screenshot 2019-11-06 at 15 11 17](https://user-images.githubusercontent.com/9249820/68384135-3cad6300-0157-11ea-97d1-c7c4180abb6c.png)

Otherwise the text will be all the way out to the edge, UX suggested 4 px.